### DISCUSSION TITLE
Ortur Rotary Parts

### DISCUSSION THREAD

**Discussion by cloudislands** [arkypita/LaserGRBL#1654] [General]
I'm hoping this forum may be a resource for my issue,  apologies if its the wrong place;  

 The parts that slide on to the ends of the rollers do not fit on the short end of one of the rollers but all fit on the other roller.  the short end of the rollers measure different sizes;  49 and 52....  please advise. I am just now attempting to assemble after purchasing months ago (at a premium price) because of the many parts it came in.   Wondering if anyone had similar issue.  I have not heard back from MFG'r.

Thank you in advance for your help.

**Comment by arkypita** [comment]
No, this is not the right place. 

LaserGRBL is a free and opern source software not related to any specific laser engraver hardware or vendors. If you have trouble with your hardware please do not write here.

I can suggest to contact ortur support via their official channel:
https://ortur.tech/support/
https://www.facebook.com/groups/orturusergroup/
